src/explore-education-statistics-common/src/components/form/FormSelect.tsx

```diff
@@ -26,9 +26,19 @@ export interface FormSelectProps
   disabled?: boolean;
   error?: string;
   id: string;
+  /**
+   * Renders the label and select inline.
+   * If a hint is provided it appears below the label
+   * and select instead of between them.
+   */
   inline?: boolean;
+  /**
+   * Renders the hint inline with the label.
+   * Has no effect if `inline` is true.
```

Comment:
Is this actually true? It looks like if `inline` and `inlineHint` are true, then the div applying the `inlineHint` styling still gets applied?

Shouldn't this be a bit more precise so that we don't add the container?

We could also add a test for this.

Reply:
Yep good point, that's sorted now.